cypress/e2e/pages/testsForAllPages.js

```diff
@@ -28,6 +29,32 @@ export const testsThatAlwaysRunForAllPages = ({
       }
     });
 
+    it('should have href values correctly formed', () => {
+      const masterbrandOverrides = [
```

Comment:
I don't think we need these overrides - these are only required in the backend to convert these weird masterbrands to a standard URL for that service